src/overlays/actors/ovl_Bg_Zg/z_bg_zg.c

```diff
@@ -108,8 +108,8 @@ void BgZg_Update(Actor* thisx, PlayState* play) {
     s32 action = this->action;
 
     if (((action < 0) || (1 < action)) || (sActionFuncs[action] == NULL)) {
-        // "Main Mode is wrong!!!!!!!!!!!!!!!!!!!!!!!!!"
-        PRINTF(VT_FGCOL(RED) "メインモードがおかしい!!!!!!!!!!!!!!!!!!!!!!!!!\n" VT_RST);
+        PRINTF(VT_FGCOL(RED) T("メインモードがおかしい!!!!!!!!!!!!!!!!!!!!!!!!!\n",
+                               "Something's wrong with main mode!!!!!!!!!!!!!!!!!!!!!!!!!\n") VT_RST);
```

Comment:
Here in update it says "Somethings wrong with the x mode" and below in draw it says "the x mode is wrong". But in japanese they are the same phrase. 
I think "the x mode is wrong" is the better translation and they should be consistent

Reply:
this happens a bunch in other files it seems